fitbenchmarking/fitbenchmarking
# Add Hessians for hellinger_nlls and poisson cost functions

#### Description of Work

Fixes #889 


#### Testing Instructions

1. Run fitbenchmarking with analytic hessians selected for both hellinger_nlls and poisson cost functions for scipy Newton-CG minimizer. Check results are comparable for this minimizer when hessians are not used.
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
